stix_shifter_modules/trustar/stix_translation/json/from_stix_map.json

```diff
@@ -0,0 +1,70 @@
+{
+  "x-trustar-query": {
```

Comment:
there's an inconsistency between the field mappings. in to_stix_map same fields are mapped to `x-trustar-report` object but here you are calling it `x-trustar-query`. I guess this object should be `x-trustar-query` as well.

have a look at other objects to make them consistent between two mapping files.

Reply:
We have implemented like : x-trustar-report